src/redux/thunks/saveResourceChanges.ts

```diff
@@ -0,0 +1,42 @@
+import {createAsyncThunk} from '@reduxjs/toolkit';
+
+import fs from 'fs/promises';
+import log from 'loglevel';
+
+import {setAutosavingError, setAutosavingStatus} from '@redux/reducers/main';
+import {selectedFilePathSelector} from '@redux/selectors';
+import {getAbsoluteFilePath} from '@redux/services/fileEntry';
+import {mergeManifests} from '@redux/services/manifest-utils';
+
+import {AppDispatch} from '@shared/models/appDispatch';
+import {RootState} from '@shared/models/rootState';
+
+export const saveResourceChanges = createAsyncThunk<void, string, {dispatch: AppDispatch; state: RootState}>(
```

Comment:
Let's rename this to `saveFormEditorResource` so we don't confuse it with something else